game/world/triggers/movetrigger.cpp

```diff
@@ -134,7 +134,8 @@ void MoveTrigger::processTrigger(const TriggerEvent& e, bool onTrigger) {
   if(moverKeyFrames.size()==0)
     return;
   if(state!=Idle) {
-    world.triggerEvent(e);
+    // can lead to infinite amount of events if multiple movers with same name exist
+    // world.triggerEvent(e);
```

Comment:
While I do agree, that this code need to be changed, but we still need to be aware of case, when 2 triggers do overlap. 
Logic here is to postpone `e` until current trigger is done execution. Maybe we can use `delayedEvent` here instead.

Reply:
Bump